specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/FormRecognizer.json

```diff
@@ -1221,7 +1300,9 @@
       "required": [
         "docType",
         "pageRange",
-        "fields"
+        "fields",
+        "predictedModel",
+        "predictedModelConfidence"
```

Comment:
Are these always required?  What happens if I'm calling analyze on a single supervised model, not a composed one?

Reply:
Even for single supervised model we still can extract confidence of document belonging to that model.
I can omit predictedModel if you think it's too excessive for single model. But confidence I think is beneficial for all users doesn't matter is it single model or composed one.

Reply:
Ah, that makes sense.

Since customers will probably think in terms of each child model recognizes a single form type, would it make sense to call predictedModel something more like predictedFormType?  If so, I think you can keep it required, and it will work for a single model in addition to a composed model.

Reply:
How is predictedModel different from docType?  Is docType unique across models?  If so, it might make more sense from a customer-perspective to get docType and docTypeConfidence -- since the guess is about what document type it is, and the model corresponds do that.  Users of the analyze APIs may not know how to map a modelId back to a document type, so getting a predicted model id may not be useful to them.